src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.ts

```diff
@@ -470,14 +471,42 @@ export class ExerciseAssessmentDashboardComponent implements OnInit, AfterViewIn
      * Calculates the status of a submission by inspecting the result
      * @param submission Submission which to check
      */
-    calculateStatus(submission: Submission) {
+    calculateSubmissionStatus(submission: Submission) {
         const tmpResult = submission.latestResult;
         if (tmpResult && tmpResult!.completionDate && Result.isManualResult(tmpResult!)) {
             return 'DONE';
         }
         return 'DRAFT';
     }
 
+    calculateComplaintStatus(complaint: Complaint) {
+        // a complaint is handled if it is either accepted or denied and a complaint response exists
+        const handled = complaint.accepted !== undefined && complaint.complaintResponse !== undefined;
+        if (handled) {
+            // handled
```

Comment:
I think you can remove that comment

Reply:
Good catch 115ba177593ecf450a2ea2fd3736e5a79e6529e1